### PR TITLE
chore(maintenance): replace moment with dayjs, condense bounty panels

### PR DESCRIPTION
**Pull request by TobiTenno**
**Summary:**
- Remove moment.js (remove large dependency from the bundle)
- Reduce padding between option and the bounty table
- Adopt new attacker/defender fields in API
- Leverage factionKey in invasions for corrected coloring
- Turn off "unknown" construction that has been empty since it was made
- Clean up date creation logic in Anomaly panel, remove beta tooltip (since it's pretty solid now)
- Show invasion % complete and ETA in progress bar

**Fixes issue (include link):** N/A
**Mockups, screenshots, evidence:** N/A

### CODE REVIEW

File: src/components/TimeBadge.vue [hunk 1]
```diff
@@ -5,7 +5,9 @@
 </template>
 
 <script>
-import moment from 'moment';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration'
+dayjs.extend(duration)
```
Comment (EricSihaoLin) [inline src/components/TimeBadge.vue:10]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121&bid=20247414)

File: src/components/panels/NewsPanel.vue [hunk 2]
```diff
@@ -91,7 +91,12 @@ body .list-group .list-group-item-borderless .list-group {
 </style>
 <script>
 import HubPanelWrap from '@/components/HubPanelWrap';
-import moment from 'moment';
+
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import updateLocale from 'dayjs/plugin/updateLocale';
+dayjs.extend(relativeTime)
```
Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue:98]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/BountyPanel.vue [hunk 3]
```diff
@@ -64,6 +64,12 @@
   </HubPanelWrap>
 </template>
 
+<style scoped>
+.list-group-item-borderless { padding-bottom: 0; }
+.list-group-item-borderbottom { padding-top: 0; }
+.table { margin-bottom: 0.5em; }
```
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue:70]: Codacy found an issue: [Replace `·margin-bottom:·0.5em;·` with `⏎··margin-bottom:·0.5em;⏎`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/TimeBadge.vue [hunk 4]
```diff
@@ -5,7 +5,9 @@
 </template>
 
 <script>
-import moment from 'moment';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration'
```
Comment (EricSihaoLin) [inline src/components/TimeBadge.vue:9]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/TimeBadge.vue (hunk 1 above)
Comment (EricSihaoLin) [inline src/components/TimeBadge.vue:10]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/ResetPanel.vue [hunk 5]
```diff
@@ -42,8 +45,8 @@ export default {
   },
   methods: {
     gettime: function() {
-      this.now = moment().toISOString();
-      this.nextDay = moment()
+      this.now = dayjs().toISOString();
+      this.nextDay = dayjs()
```
Comment (EricSihaoLin) [inline src/components/panels/ResetPanel.vue:49]: Codacy found an issue: [Replace `⏎········.utc()⏎········.endOf('day')⏎········.add(1,·'seconds')⏎········` with `.utc().endOf('day').add(1,·'seconds')`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/TimeBadge.vue (hunk 4 above)
Comment (EricSihaoLin) [inline src/components/TimeBadge.vue:9]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/ResetPanel.vue [hunk 6]
```diff
@@ -26,8 +29,8 @@ export default {
   },
   data() {
     return {
-      now: moment().toISOString(),
-      nextDay: moment()
+      now: dayjs().toISOString(),
+      nextDay: dayjs()
```
Comment (EricSihaoLin) [inline src/components/panels/ResetPanel.vue:33]: Codacy found an issue: [Replace `⏎········.utc()⏎········.endOf('day')⏎········.add(1,·'seconds')⏎········` with `.utc().endOf('day').add(1,·'seconds')`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/NewsPanel.vue (hunk 2 above)
Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue:98]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/NewsPanel.vue [hunk 7]
```diff
@@ -91,7 +91,12 @@ body .list-group .list-group-item-borderless .list-group {
...
+dayjs.extend(updateLocale)
```
Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue:99]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue:99]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/BountyPanel.vue [hunk 8]
```diff
@@ -64,6 +64,12 @@
   </HubPanelWrap>
 </template>
 
+<style scoped>
+.list-group-item-borderless { padding-bottom: 0; }
```
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue:68]: Codacy found an issue: [Replace `·padding-bottom:·0;·` with `⏎··padding-bottom:·0;⏎`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/BountyPanel.vue [hunk 9]
```diff
@@ -64,6 +64,12 @@
...
+.list-group-item-borderbottom { padding-top: 0; }
```
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue:69]: Codacy found an issue: [Replace `·padding-top:·0;·` with `⏎··padding-top:·0;⏎`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121&bid=20247414)

File: src/components/TimeBadge.vue (hunk 1 above)
Comment (EricSihaoLin) [inline src/components/TimeBadge.vue]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/NewsPanel.vue (hunk 2 above)
Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/BountyPanel.vue (hunk 3 above)
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue]: Codacy found an issue: [Replace `·margin-bottom:·0.5em;·` with `⏎··margin-bottom:·0.5em;⏎`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/TimeBadge.vue (hunk 4 above)
Comment (EricSihaoLin) [inline src/components/TimeBadge.vue]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/ResetPanel.vue (hunk 5 above)
Comment (EricSihaoLin) [inline src/components/panels/ResetPanel.vue:49]: Codacy found an issue: [Replace `⏎········.utc()⏎········.endOf('day')⏎········.add(1,·'seconds')⏎········` with `.utc().endOf('day').add(1,·'seconds')`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/ResetPanel.vue (hunk 6 above)
Comment (EricSihaoLin) [inline src/components/panels/ResetPanel.vue:33]: Codacy found an issue: [Replace `⏎········.utc()⏎········.endOf('day')⏎········.add(1,·'seconds')⏎········` with `.utc().endOf('day').add(1,·'seconds')`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/NewsPanel.vue (hunk 2 above)
Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/NewsPanel.vue (hunk 7 above)
Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

Comment (EricSihaoLin) [inline src/components/panels/NewsPanel.vue]: Codacy found an issue: [Insert `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/BountyPanel.vue (hunk 8 above)
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue]: Codacy found an issue: [Replace `·padding-bottom:·0;·` with `⏎··padding-bottom:·0;⏎`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

File: src/components/panels/BountyPanel.vue (hunk 9 above)
Comment (EricSihaoLin) [inline src/components/panels/BountyPanel.vue]: Codacy found an issue: [Replace `·padding-top:·0;·` with `⏎··padding-top:·0;⏎`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6208121)

**Comment by TobiTenno** [comment]
aaaaand codacy decided to derp again... it's not even matching linting results at this point...

**Review by EricSihaoLin — APPROVED** [review]

**Comment by TobiTenno** [comment]
I found a smol issue with counters that i'm fixing
noticed it on the invasion hover stuff. also adding a requested feature on the invasion bars

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**